apps/project/models.py

```diff
@@ -186,6 +186,10 @@ def get_for_gq(cls, user, only_member=False):
             return visible_projects.filter(current_user_role__isnull=False)
         return visible_projects
 
+    @classmethod
+    def get_for_public_project(cls):
```

Comment:
```suggestion
    def get_public_projects(cls):
```

But this is not used?